Formula/dps8m.rb

```diff
@@ -0,0 +1,31 @@
+class Dps8m < Formula
+  desc "Simulator for the Multics dps-8/m mainframe"
+  homepage "http://ringzero.wikidot.com"
+  url "https://downloads.sourceforge.net/project/dps8m/Release%201.0/source.tgz"
+  version "1.0"
+  sha256 "51088dd91de888b918644c431eec22318640d28eb3050d9c01cd072aa7cca3c7"
+
+  depends_on "libuv"
+
+  def install
+    bin.mkpath
+    system "make", "INSTALL_ROOT=#{prefix}", "install"
+  end
+
+  test do
+    (testpath/"test.exp").write <<-EOS.unindent.undent
+      spawn #{bin}/dps8
+      expect "DPS8/M emulator (git b7a50ffc)
```

Comment:
The git commit (b7a50ffc) will change every release so that's not something you'll want to expect.